deployability/modules/testing/tests/test_central_components/test_install.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import pytest
+
+from modules.testing.utils import logger
+from ..helpers.constants import WAZUH_ROOT
+from ..helpers.executor import WazuhAPI
+from ..helpers.generic import HostConfiguration, HostInformation, GeneralComponentActions
+from ..helpers.manager import WazuhManager
+from ..helpers.indexer import WazuhIndexer
+from ..helpers.dashboard import WazuhDashboard
+from ..helpers.central import WazuhCentralComponents
+from ..helpers.utils import Utils
+
+
+@pytest.fixture(scope="module", autouse=True)
+def wazuh_params(request):
+    wazuh_version = request.config.getoption('--wazuh_version')
+    wazuh_revision = request.config.getoption('--wazuh_revision')
+    dependencies = request.config.getoption('--dependencies')
+    targets = request.config.getoption('--targets')
+
+    return {
+        'wazuh_version': wazuh_version,
+        'wazuh_revision': wazuh_revision,
+        'dependencies': dependencies,
+        'targets': targets
+    }
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_test_environment(wazuh_params):
+    targets = wazuh_params['targets']
+    # Clean the string and split it into key-value pairs
+    targets = targets.replace(' ', '')
+    targets = targets.replace('  ', '')
+    pairs = [pair.strip() for pair in targets.strip('{}').split(',')]
+    targets_dict = dict(pair.split(':') for pair in pairs)
+
+    wazuh_params['master'] = targets_dict.get('wazuh-1')
+    wazuh_params['workers'] = [value for key, value in targets_dict.items() if key.startswith('wazuh-') and key != 'wazuh-1']
+    wazuh_params['indexers'] = [value for key, value in targets_dict.items() if key.startswith('node-')]
+    wazuh_params['dashboard'] = targets_dict.get('dashboard', wazuh_params['master'])
+
+    # If there are no indexers, we choose wazuh-1 by default
+    if not wazuh_params['indexers']:
+        wazuh_params['indexers'].append(wazuh_params['master'])
+
+    wazuh_params['managers'] = {key: value for key, value in targets_dict.items() if key.startswith('wazuh-')}
+
+
+def test_installation(wazuh_params):
+    # Disabling firewall for all managers
+    for _, manager_params in wazuh_params['managers'].items():
+        Utils.check_inventory_connection(manager_params)
+        HostConfiguration.disable_firewall(manager_params)
+
+    # Certs create and scp from master to worker
+    HostConfiguration.certs_create(wazuh_params['wazuh_version'], wazuh_params['master'], wazuh_params['dashboard'], wazuh_params['indexers'], wazuh_params['workers'])
+
+    # Install central components and perform checkfile testing
+    for _, manager_params in wazuh_params['managers'].items():
+        #WazuhCentralComponents.perform_install_and_scan_for_aio(manager_params, wazuh_params)
```

Comment:
Remove comment

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5324/commits/96c47257f1bca33c7f144285a4c7b63a8c10fbce